exporter/debugexporter/internal/otlptext/logs.go

```diff
@@ -38,6 +38,9 @@ func (textLogsMarshaler) MarshalLogs(ld plog.Logs) ([]byte, error) {
 				buf.logEntry("Timestamp: %s", lr.Timestamp())
 				buf.logEntry("SeverityText: %s", lr.SeverityText())
 				buf.logEntry("SeverityNumber: %s(%d)", lr.SeverityNumber(), lr.SeverityNumber())
+				if lr.EventName() != "" {
```

Comment:
can you ensure a test for this in https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/debugexporter/internal/otlptext/logs_test.go

Reply:
Added!